website/react-magma-docs/src/pages/api/popover.mdx

```diff
@@ -54,12 +55,52 @@ export function Example() {
 }
 ```
 
+## Inverse
```

Comment:
Sorry to nitpick, but the inverse example is towards the end of almost every other component. Could we move it down to be the last example?